staff/miguel-angel-sevillano/CV/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Miguel Angel Sevillano CV</title>
+</head>
+
+<body>
+    <section>
+        <h1>Personal info</h1>
+        <article>
+            <b>Name:</b> Miguel Angel.<br>
+            <b>Address:</b> Barcelona.<br>
+            <b>Age:</b> 34.<br>
+            <b>Contac:</b> mash5550@hotmail.com<br>
+        </article>
+    </section>
+    <section>
+        <h1>Academic formation</h1>
+        <table>
+            <tr>
```

Comment:
@MiguiHidalgo usually a `<table>` contains `<tr>`with `<th>`headers and '<td>`data. if it is not your case, then it's fine, just fyi.